lib/database/schemas.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+/**
+ * Individual Content Type Schema
+ */
+const content = [
+  {
+    name: 'revision',
+    type: 'increments',
+    index: true,
+  },
+  {
+    name: 'id',
+    type: 'string',
+  },
+  {
+    name: 'created',
+    type: 'timestamp',
+  },
+  {
+    name: 'language',
+    type: 'string',
+  },
+  {
+    name: 'sunrise',
```

Comment:
Probably inside `lib/content-types`. They aren't schema. The user mocking will live inside `lib/auth` or `lib/user` or something like that.